src/django_components/compat/django.py

```diff
@@ -0,0 +1,116 @@
+from argparse import ArgumentParser
+from typing import Any, Optional, Type
+
+import django
+from django.core.management.base import BaseCommand as DjangoCommand
+
+from django_components.util.command import (
```

Comment:
In this file is logic that converts the `ComponentCommand` into Django's `BaseCommand`.